src/config.rs

```diff
@@ -2955,6 +2953,14 @@ impl TiKvConfig {
         // Reserve some space for page cache. The
         ReadableSize((total as f64 * MEMORY_USAGE_LIMIT_RATE) as u64)
     }
+
+    pub fn configure_shared_rocks_env(&self, env: &Env) {
+        if !self.raft_engine.enable {
+            env.set_high_priority_background_threads(
```

Comment:
I suggest to add some explanation where the changes to the TiKV based on internal logic of rocksdb, which is convenient for maintenance.